src/hooks/use-suggestions.ts

```diff
@@ -239,6 +240,16 @@ export const useSuggestions = (
             )
             if (!dataType) {
                 localSuggestions = []
+                // 2nd half of key aliases with the dot: matchedField ~ 'Item.H' of 'Item.Height'
+                const matchedFieldWithDot =
+                    matchedField.indexOf('.') > 0
+                        ? matchedField
+                        : matchedField + '.'
+                aliasesArray.forEach((a) => {
```

Comment:
use a for-of loop instead of foreach please

Reply:
done